Portfolio/index.html

```diff
@@ -1,16 +1,72 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Portfolio</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
-    </div>
-    <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
-    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
-  </body>
-</html>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Portfolio</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="index.css" />
+  </head>
+
+  <body style="margin: 0; padding: 0">
+    <div top="0">
+      <a href="../index.html">Go Back</a>
+    </div>
+    <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
+    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
+    <header>
+        <h1>Priyanka  Kumari</h1>
+        <img src="https://i.pinimg.com/736x/de/6f/21/de6f21a62c5414ec1f65b41afc173539.jpg" alt="priyanka-photo" width="150" >
+        <p>Web Developer & Software Developer</p>
+    </header>
+
+    <section>
+        <h2>About me</h2>
+        <p>Welcome to my portfolio.... <br> I am priyanka kumari,I'm web-developer and software developer ,I doing code in html and css as well javaScript and react
+           I'm complited 10th or 12th from bihar bord. I love doing dance and yoga it is I'm doing when I found free from our work I love traveling and advanturing diffrent diffren place
+           .About my family My father is farmer and my mother is house-wife .I've 4 siblings 3 sister and 1 brother ,I'm elder in my siblings
+        </p>
+    </section>
+
+    <section>
+        <h2>About my Skill</h2>
+        <p>I've skill like..</p>
+        <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>javaScript</li>

```

Comment:
The form's 'action' attribute should be set to an appropriate endpoint, not an email address. Consider using a server-side script or API to handle form submissions.